requirements/dev.txt

```diff
@@ -1,10 +1,10 @@
 -r base.txt
 flake8==3.3.0
-flake8_docstrings==1.0.3
+flake8-docstrings==1.1.0
 django-debug-toolbar==1.7
 freezegun==0.3.8
 isort==4.2.5
-pydocstyle==1.1.1 # pyup: >=1.1.1,<2.0
```

Comment:
we could drop the explicit pydocstyle requirement from our project as it will be pulled in from flake8-docstrings automatically